CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+## 0.1.0 (2021-05-07)
+
+This is the first public Alpha release of the Realm Kotlin SDK for Android and Kotlin Multiplatform.
+
+A minimal setup for including this library looks like this:
+
+```
+// Top-level build.gradle file
+buildscript {
```

Comment:
I'm looking into the marker artifact now as part of the Gradle Plugin Portal work. But it pushes the marker artifact to an entirely new location `/realm-kotlin/....` instead of `/io/realm/kotlin/...` so I need to get a hold of Maven Central support first so we can push artifacts to that location.